backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -242,6 +242,17 @@ describe("Listings", () => {
     expect(modifiedListing.events[0].file.label).toBe(listingEvent.file.label)
   })
 
+  it("should sort results from most recently updated to least", async () => {
+    const res = await supertest(app.getHttpServer()).get(`/listings?limit=all`).expect(200)
+    for (let i = 0; i < res.body.items.length - 1; ++i) {
+      const currentUpdatedAt = new Date(res.body.items[i].updatedAt)
+      const nextUpdatedAt = new Date(res.body.items[i + 1].updatedAt)
+
+      // Verify that each listing's updatedAt timestamp is more recent than the next listing's.
+      expect(currentUpdatedAt.getTime()).toBeGreaterThan(nextUpdatedAt.getTime())
```

Comment:
Is it worth making another request in this test for the next page, to make sure results are ordered correctly across pages, or does the unit test in listings.spec.ts cover that well enough?

Reply:
I like this! Added a second test for the interaction of pagination and sorting